demos/security/README.md

```diff
@@ -0,0 +1,101 @@
+# IPSec demo
+
+## hardware
+
+- server with Ubuntu 22.04 
+- Nvidia BlueField2
+- Keysight CloudStorm 100G
+
+## configuration
+
+### host (the server holding the DPU)
+
+- install Ubuntu 22.04 server
+- install `MLNX_OFED_LINUX-5.8-1.1.2.1-ubuntu22.04-x86_64.iso`
+- install `DOCA_1.5.1_BSP_3.9.3_Ubuntu_20.04-4.2211-LTS.signed.bfb` on the BlueField2
+- set BlueField2 in SEPARATED_HOST mode to make things easier
+  ```
+  mlxconfig -d /dev/mst/mt41686_pciconf0 s INTERNAL_CPU_MODEL=0
+  mlxconfig -d /dev/mst/mt41686_pciconf0.1 s INTERNAL_CPU_MODEL=0
+  ```
+
+### dpu (BlueField2)
+
+- enable ip forwarding `sysctl net.ipv4.ip_forward=1`
+- set ip addresses on the phisical links
+  ```
+  ip addr add 200.0.0.1/24 dev enp3s0f0s0
+  ip addr add 201.0.0.1/24 dev enp3s0f1s0
+  ```
+- install Docker (see [Docker manual](https://docs.docker.com/engine/install/ubuntu/) )
+- start OPI server
+```
+git clone https://github.com/opiproject/opi-strongswan-bridge
+cd ./opi-strongswan-bridge
+
+docker pull ghcr.io/opiproject/opi-security-server:main
+
+sudo sed -i "s/# socket = unix:\/\/\${piddir}\/charon\.vici/socket = unix:\/\/\/var\/run\/charon\.vici/g" /etc/strongswan.d/charon/vici.conf
+
+docker run --rm --network host \
+    --mount src=/var/run,target=/var/run,type=bind \
+    -d ghcr.io/opiproject/opi-security-server:main /opi-vici-bridge -port=50151
+
+
+
+# debug
+docker run --rm --network host \
+    --mount src=/var/run,target=/var/run,type=bind \
+    -it ghcr.io/opiproject/opi-security-server:main sh
+
+https://github.com/opiproject/opi-strongswan-bridge/blob/main/conf/strongswan-server.conf#L21
```

Comment:
Should this be a comment?

Reply:
i removed it i, went that path till i noticed they were already built so there was no need to build myself teh python lib